shex/geneticDiagnosisShape.shex

```diff
@@ -1,25 +1,96 @@
-PREFIX : <http://purl.org/ejp-rd/cde/v020/shex/>
+PREFIX : <http://purl.org/ejp-rd/cde/v030/shex/>
 PREFIX obo: <http://purl.obolibrary.org/obo/> 
 PREFIX sio: <http://semanticscience.org/resource/>
-PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 
-:processShape IRI { 
-  a [obo:NCIT_C15709] ;
-  sio:SIO_000229 @:variantShape 
+:identifierShape IRI {
+    a [sio:SIO_000115] ;
+    sio:SIO_000020 @:geneticRoleShape ;
+    sio:SIO_000300 xsd:string
 }
 
 :personShape IRI { 
-  a [sio:SIO_000498] ;
-  sio:SIO_000228 @:roleShape 
+    a [sio:SIO_000498] ;
+    sio:SIO_000228 @:geneticRoleShape ;
+    sio:SIO:000008 @:hgvsAttributeShape ;
+    sio:SIO:000008 @:omimAttributeShape
```

Comment:
I think you miss @:hgncAttributeShape here.

Reply:
Done